submitted_models/cerberus_anymal_c_sensor_config_1/model.sdf

```diff
@@ -0,0 +1,1772 @@
+              <max_angle>3.1459</max_angle>
+            </horizontal>
+            <vertical>
+              <samples>16</samples>
+              <resolution>1</resolution>
+              <min_angle>-0.261799</min_angle>
+              <max_angle>0.261799</max_angle>
+            </vertical>
+          </scan>
+          <range>
+            <min>0.05</min>
+            <max>100</max>
+            <resolution>0.01</resolution>
+          </range>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.01</stddev>
+          </noise>
+        </lidar>
+        <pose frame="">-0.364 0.000 0.142 0.000 0.000 -1.571</pose>
+      </sensor>
+      <sensor name="front_bpearl" type="gpu_lidar">
+        <visualize>0</visualize>
+        <update_rate>10</update_rate>
+        <lidar>
+          <scan>
+            <horizontal>
+              <samples>440</samples>
+              <resolution>1</resolution>
+              <min_angle>-3.1459</min_angle>
+              <max_angle>3.1459</max_angle>
+            </horizontal>
+            <vertical>
+              <samples>32</samples>
+              <resolution>1</resolution>
+              <min_angle>0.040360739</min_angle>
+              <max_angle>1.562069681</max_angle>
+            </vertical>
+          </scan>
+          <range>
+            <min>0.3</min>
+            <max>31</max>
+            <resolution>0.001</resolution>
+          </range>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.008</stddev>
+          </noise>
+        </lidar>
+        <pose frame="">0.524 0.000 -0.042 -2.576 0.245 -1.936</pose>
+      </sensor>
+      <sensor name="rear_bpearl" type="gpu_lidar">
+        <visualize>0</visualize>
+        <update_rate>10</update_rate>
+        <lidar>
+          <scan>
+            <horizontal>
+              <samples>440</samples>
+              <resolution>1</resolution>
+              <min_angle>-3.1459</min_angle>
+              <max_angle>3.1459</max_angle>
+            </horizontal>
+            <vertical>
+              <samples>32</samples>
+              <resolution>1</resolution>
+              <min_angle>0.040360739</min_angle>
+              <max_angle>1.562069681</max_angle>
+            </vertical>
+          </scan>
+          <range>
+            <min>0.3</min>
+            <max>31</max>
+            <resolution>0.001</resolution>
+          </range>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.008</stddev>
+          </noise>
+        </lidar>
+        <pose frame="">-0.524 -0.000 -0.042 -2.576 0.245 1.206</pose>
+      </sensor>
+      <sensor name="camera_0" type="camera">
+        <always_on>1</always_on>
+        <update_rate>20</update_rate>
+        <pose frame="">-0.400 -0.050 0.229 -0.006 0.011 3.133</pose>
+        <camera name="camera_0">
+          <horizontal_fov>2.19911</horizontal_fov>
```

Comment:
@mjcarroll noticed that these cameras don't have lens properties (specified in the `<lens>` tag). Can you please add them?

Reply:
Using our standard formulae, I would recommend:

```
<intrinsics>
    <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
    <fx>183.43</fx>
    <fy>183.43</fy>
    <!-- cx = ( width + 1 ) / 2 -->
    <cx>360.5</cx>
    <!-- cy = ( height + 1 ) / 2 -->
    <cy>270.5</cy>
    <s>0</s>
</intrinsics>
```

Reply:
@Samu24 can you please follow mjcarroll's suggested lens intrinsic parameters?  I believe this is the last outstanding issue prior to acceptance.

Reply:
Yes I can take care of that.

Reply:
@acschang @azeey
Lens properties have been added.